src/crudeditor-lib/middleware/notifications.js

```diff
@@ -145,7 +140,29 @@ const eventsMiddleware = ({ context, modelDefinition }) => store => next => acti
     default:
   }
 
-  return next(action)
+  if ([
+    CREATE_ALL_INSTANCE_FIELDS_VALIDATE,
+    EDIT_ALL_INSTANCE_FIELDS_VALIDATE
+  ].indexOf(action.type) > -1) {
+    const currentView = store.getState().common.activeViewName;
```

Comment:
What's the point of calculating **currentView** *before* and other variables *after* next(action) call?  It only causes unneeded Qs by the one doing code maintenance.